docs/Ecosystem.md

```diff
@@ -5,13 +5,20 @@ title: "Ecosystem"
 
 # Ecosystem
 
-- [Inputs and Fields](#inputs-and-fields)
-- [Translations](#translations)
-- [Authentication Providers](#authentication-providers)
-- [Authorization Management](#authorization-management)
-- [Data Providers](#data-providers)
-- [User Interface](#user-interface)
-- [Miscellaneous](#miscellaneous)
+- [Ecosystem](#ecosystem)
```

Comment:
Why did you add a link to the page itself?

Reply:
I don't even remember adding it ^^